util/design/gen-otp-immutable-rom-ext-json.py

```diff
@@ -107,6 +149,10 @@ def main():
                         type=str,
                         metavar='<path>',
                         help='Output JSON file path.')
+    parser.add_argument('--hardened-true-value',
```

Comment:
I think this parameter is a bit overkill. The truth value for enablement is hardcoded in the ROM already, so I think we can hardcode in this script too, and just drop a comment saying it must the value in the ROM: https://github.com/lowRISC/opentitan/blob/05db1550680f82a5d555e402ad4454edf82a0228/sw/device/silicon_creator/rom/rom.c#L673

Reply:
okay, done.